packages/grid/_modules_/grid/components/columnHeaders/GridColumnHeaderItem.tsx

```diff
@@ -64,22 +68,6 @@ export const GridColumnHeaderItem = ({
     });
   }
 
-  const onDragStart = React.useCallback(
-    (event) => apiRef!.current.onColItemDragStart(column, event.currentTarget),
-    [apiRef, column],
-  );
-  const onDragEnter = React.useCallback((event) => apiRef!.current.onColItemDragEnter(event), [
-    apiRef,
-  ]);
-  const onDragOver = React.useCallback(
-    (event) =>
-      apiRef!.current.onColItemDragOver(column, {
-        x: event.clientX,
-        y: event.clientY,
-      }),
-    [apiRef, column],
-  );
-
   const publish = React.useCallback(
     (eventName: string) => (event: React.MouseEvent) =>
       apiRef!.current.publishEvent(
```

Comment:
You need to publish the event with params. For cell, we have `GridCellParams`, could you add `GridColumnHeaderParams` please